### PR TITLE
Feature. Docker secrets enabled.

### LINKED ISSUE #12: Infra: Prepare spring profile for production with MySQL settings, store secrets in Docker Swarm
* profile `production` with settings for MySQL connection
* Secrets should be added to Docker Swarm on the box
* Secrets should be granted to services via `docker-compose.yaml`
secrets in docker swarm: https://docs.docker.com/engine/swarm/secrets/ (more precisely, [usage in compose file](https://docs.docker.com/engine/swarm/secrets/#use-secrets-in-compose))

### PR DESCRIPTION
**Pull request by aktsay6**
### What's done: 
  * Added secrets to server
  * Fixed bugs with liquibase
This pull request closes #12

### CODE REVIEW

File: docker-compose.yaml.template [hunk 1]
```diff
@@ -51,4 +56,11 @@ services:
     ports:
       - "9100:3000"
 volumes:
-  save:
\ No newline at end of file
+  save:
+  db_data:
```
Comment (petertrr) [inline docker-compose.yaml.template:60]: What is this?

File: save-backend/src/main/kotlin/org/cqfn/save/backend/postprocessor/DockerSecretsDatabaseProcessor.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        if (passwordResource.exists()) {
+            try {
+                val dbPassword = StreamUtils.copyToString(passwordResource.inputStream, Charset.defaultCharset())
+                val dbUsername = StreamUtils.copyToString(usernameResource.inputStream, Charset.defaultCharset())
+                val props = Properties()
+                props["spring.datasource.password"] = dbPassword
+                props["spring.datasource.username"] = dbUsername
+                environment?.propertySources?.addLast(PropertiesPropertySource("dbProps", props))
+            } catch (exc: IOException) {
+                throw RuntimeException(exc)
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/postprocessor/DockerSecretsDatabaseProcessor.kt:30]: Why are you wrapping it like this?

**Review by petertrr — CHANGES_REQUESTED** [review]

File: build.gradle.kts [hunk 3]
```diff
@@ -40,7 +40,7 @@ liquibase {
         register("main") {
             arguments = mapOf(
                     "changeLogFile" to "db/changelog/db.changelog-master.xml",
-                    "url" to "jdbc:mysql://$databaseUrl/save_cloud",
```
Comment (petertrr) [inline build.gradle.kts:43]: Also let's move `secretes` file to home dir

File: build.gradle.kts [hunk 4]
```diff
@@ -18,8 +18,8 @@ val profile = properties.getOrDefault("profile", "dev") as String
 val props = java.util.Properties()
 val file = file("save-backend/src/main/resources/application-$profile.properties").apply { props.load(inputStream()) }
 
-if (File("secrets").exists()) {
-    file("secrets").apply { props.load(inputStream()) }
+if (File("../secrets").exists()) {
```
Comment (petertrr) [inline build.gradle.kts:21]: Use `HOME` env variable here

File: docker-compose.yaml.template [hunk 5]
```diff
@@ -18,7 +18,12 @@ services:
   backend:
     image: 127.0.0.1:6000/save-backend:{{project.version}}
     environment:
-      - SPRING_PROFILES_ACTIVE={{profile}}
+      - "SPRING_PROFILES_ACTIVE={{profile}}"
+      - "MYSQL_USER: /run/secrets/db_username"
```
Comment (petertrr) [inline docker-compose.yaml.template:22]: Is there any difference in `=` and `:` syntax? If not, it'd be better to keep it consistent

File: save-backend/src/main/resources/application-dev.properties [hunk 6]
```diff
@@ -8,6 +8,6 @@ spring.jpa.properties.hibernate.validator.apply_to_ddl=false
 spring.liquibase.change-log=db/changelog/db.changelog-master.xml
```
Comment (petertrr) [inline save-backend/src/main/resources/application-dev.properties:8]: `test/resources/application.properties`, it won't be needed in `dev` profile

File: docker-compose.yaml.template (hunk 1 above)
Comment (aktsay6) [reply docker-compose.yaml.template]: Fixed

File: save-backend/src/main/kotlin/org/cqfn/save/backend/postprocessor/DockerSecretsDatabaseProcessor.kt (hunk 2 above)
Comment (aktsay6) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/postprocessor/DockerSecretsDatabaseProcessor.kt]: Fixed

File: build.gradle.kts (hunk 3 above)
Comment (aktsay6) [reply build.gradle.kts:43]: Fixed

**Review by petertrr — APPROVED** [review]

**Pull request merged**